tasks:
  task001:
    direction: 1.0
    nav_goal: {x: 5.349999883770943, y: 4.149999891594052, yaw: 3.141592653589793}
  task002:
    direction: 1.0
    nav_goal: {x: 4.749999897181988, y: 4.149999891594052, yaw: 3.141592653589793}
  task003:
    direction: 1.0
    nav_goal: {x: 4.149999910593033, y: 4.149999891594052, yaw: 3.141592653589793}
  task004:
    direction: 1.0
    nav_goal: {x: 3.549999924004078, y: 4.149999891594052, yaw: 3.141592653589793}
  task005:
    direction: 1.0
    nav_goal: {x: 2.949999937415123, y: 4.149999891594052, yaw: 3.141592653589793}
  task006:
    direction: 1.0
    nav_goal: {x: 2.329999951273203, y: 4.149999891594052, yaw: 3.141592653589793}
  task007:
    direction: 1.0
    nav_goal: {x: 0.8499999966472387, y: 3.5099999248981475, yaw: -1.5707963267948963}
  task008:
    direction: 1.0
    nav_goal: {x: 0.8499999966472387, y: 2.9099999383091926, yaw: -1.5707963267948963}
  task009:
    direction: 1.0
    nav_goal: {x: 0.889999995753169, y: 2.3099999517202376, yaw: -1.5707963267948963}
  task010:
    direction: 1.0
    nav_goal: {x: 0.889999995753169, y: 1.7099999651312827, yaw: -1.5707963267948963}
  task011:
    direction: 1.0
    nav_goal: {x: 0.9299999948590993, y: 1.0699999794363975, yaw: -1.5707963267948963}
